src/blocks/button/Button.tsx

```diff
@@ -46,7 +49,15 @@ const StyledButton = styled.button<ButtonProps>`
   }
```

Comment:
cursor: ${(props) => (props.disabled || props.loading ? 'not-allowed' : 'pointer')};